### PR TITLE
test code for review to use mechanize

### PR DESCRIPTION
**Pull request by tomato1976**

### CODE REVIEW

File: lib/tasks/crawl.rake [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    #   # sleep(rand(10)+1)
+    #   # detailpage = agent.get(link.href)
+      
+    #   # p link.text
+    #   # p link.href
+    # end
+    
+    #detailpage = agent.get(page.links_with(:href => /code/).first.href)
+    detailpage = agent.get("http://www.sakataseed.co.jp/product/search/code00459305.html")
+
```
Comment (tomato1976) [inline lib/tasks/crawl.rake]: This is just test code to crawl sites.
In my development, I have to use a cache of crawling web site to reduce load of a crwaling server.
But I have no idea now.
Do you have a good idea?

File: lib/tasks/crawl.rake [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    contents_src = contents_img.attribute('src').value
+    # => "file/459305_p.jpg"
+    contents_alt = contents_img.attribute('alt').value
+    # => "イチゴ 伊豆っ娘 （苗）"
+    contents_infos = detailpage.search("//*[@id='dContentsSearchColR']/table/tr")
+    detail_right = Hash.new
+    contents_infos.each do |i|
+      detail_right.store(i.search("th").inner_text, i.search("td").inner_text)
+    end
+
```
Comment (tomato1976) [inline lib/tasks/crawl.rake]: I crawled one detail page and I want to save this data in my database to reuse my web and to make Model.
When we want to crawl two sites, perhaps there are two format of detail pages each sites. In this case, I think two tables, models  need to save. Is it right?

Comment (rizwanreza) [inline lib/tasks/crawl.rake]: You should probably grab the data in such a way that it doesn't need two different models.

File: lib/tasks/crawl.rake (hunk 1 above)
Comment (rizwanreza) [inline lib/tasks/crawl.rake]: Look at the VCR gem: https://github.com/vcr/vcr

**Comment by tomato1976** [comment]
Sorry, I did't change my branch after your previous review.

I have some questions.
No.1  Today I have a heroku error.  I haven't solve it yet.
```
PG::UndefinedTable: ERROR:  relation "companies" does not exist
: ALTER TABLE "products" ADD CONSTRAINT "fk_rails_9fdb94e13d"
FOREIGN KEY ("company_id")
  REFERENCES "companies" ("id")
```


No.2  If I want to apply this site to multi languages, Is there a good solution?   
  
No.3 I want to make a tail way(?) (I don't know how to explain pankuzu in Japanese)  
  For example,   
    I watch a top-page and click a category link. 
   I watch the category page and click a product link.
   I watch the product page and this page have a pankuzu links like below.
        Top > XXXX category >  this page   

  Do you have a good solution.  
  
No.4  I want to make a search panel for products.  
  Can I search some columns at the same input value?

**Comment by rizwanreza** [comment]
For number 2, read this: http://guides.rubyonrails.org/i18n.html

**Comment by rizwanreza** [comment]
In English, Pankuzu is called Breadcrumb: https://www.ruby-toolbox.com/categories/Breadcrumb_Builders

**Comment by rizwanreza** [comment]
You can use something like `params[:query]` to search different fields like this: `Post.where('body LIKE %?% OR title LIKE %?%', params[:query], params[:query])`

**Pull request merged**